source/wiz/graph.py

```diff
@@ -108,6 +108,15 @@ def __init__(self, definition_mapping):
         # used as it is a FIFO queue.
         self._conflicts = collections.deque()
 
+        # Keep track of whether one or several packages are added to the graph
+        # during conflict resolution loop.
+
+        # Keep track of whether the conflict list needs to be sorted according
+        # to the latest distance mapping computed. It should always be true for
+        # the first conflict resolution loop, and be updated depending on
+        # whether the conflict list is updated.
+        self._conflicts_needs_sorting = True
```

Comment:
this is not part of any release note. what is this for? is this part of the version bug?

Reply:
This is part of the second release note. Do you think that much detail is needed in the notes?